tooling/electron-mksnapshot/package.json

```diff
@@ -6,13 +6,12 @@
   "main": "dist/mksnapshot.js",
   "scripts": {
     "build": "tsc",
-    "check-ts": "tsc --noEmit && yarn -s tslint",
+    "check-ts": "tsc --noEmit",
```

Comment:
Are these no longer needed? Or were they no-ops?

Reply:
yeah the tslint targets in these packages never worked. The pointed to a tslint.json file that didn't exist and if you ran them they would fail